app/src/main/java/org/ole/planet/myplanet/ui/team/AdapterTeamList.java

```diff
@@ -102,8 +104,10 @@ private void showActionButton(boolean isMyTeam, RecyclerView.ViewHolder holder,
         if (isMyTeam) {
             ((ViewHolderTeam) holder).action.setText("Leave");
             ((ViewHolderTeam) holder).action.setOnClickListener(view -> {
-                list.get(position).leave(user, mRealm);
-                notifyDataSetChanged();
+                new AlertDialog.Builder(context).setMessage("Are you sure want to leave this team ??").setPositiveButton("Yes", (dialogInterface, i) -> {
```

Comment:
"Are you sure you want to leave this team?" instead of "Are you sure want to leave this team ??"